taohash/validator/validator.py

```diff
@@ -155,13 +204,12 @@ def evaluate_miner_share_value(self) -> None:
                         continue
 
                     uid = hotkey_to_uid[metric.hotkey]
-                    share_value = metric.get_share_value(btc_price, btc_difficulty)
+                    share_value = metric.get_share_value_fiat(btc_price, btc_difficulty)
```

Comment:
A bit extra and may require some refactoring. But we should price everything in Bitcoin and then TAO. Remove the fiat-based calculation